Sources/Workspace/ManagedArtifact.swift

```diff
@@ -37,14 +25,19 @@ public final class ManagedArtifact {
     /// The source of the artifact (local or remote).
     public let source: Source
 
+    /// The path of the artifact on disk
+    public let path: AbsolutePath
```

Comment:
I like that no matter where it came from, there is now always a straightforward way to get to the artifact on disk.